app/routes.py

```diff
@@ -1,2 +1,257 @@
-from flask import Blueprint
+from flask import Blueprint, jsonify, make_response, request
+from app.models.task import Task
+from app.models.goal import Goal
+from app import db
+from datetime import datetime
+import requests
+import os
 
+
+tasks_bp = Blueprint("tasks", __name__, url_prefix="/tasks")
+goals_bp = Blueprint("goals", __name__, url_prefix="/goals")
+path = "https://slack.com/api/chat.postMessage"
+token = os.environ.get("SLACK_TOKEN")
+
+# T A S K S   E N D P O I N T S
+
+
+@tasks_bp.route("", methods=["GET", "POST"])
+def handle_tasks():
+    if request.method == "GET":
+        sort_query = request.args.get("sort")
+        if sort_query == "asc":
+            tasks = Task.query.order_by(Task.title.asc())
+        elif sort_query == "desc":
+            tasks = Task.query.order_by(Task.title.desc())
+        else:
+            tasks = Task.query.all()
+        tasks_response = []
+        for task in tasks:
+            tasks_response.append(
+                {
+                    "id": task.task_id,
+                    "title": task.title,
+                    "description": task.description,
+                    "is_complete": False if task.completed_at == None else True
+                }
+            )
+        return jsonify(tasks_response), 200
+
+    elif request.method == "POST":
+        request_body = request.get_json()
+        if "title" not in request_body or "description" not in request_body or "completed_at" not in request_body:
+            return {"details": "Invalid data"}, 400
+        new_task = Task(title=request_body["title"],
+                        description=request_body["description"],
+                        completed_at=request_body["completed_at"])
+        db.session.add(new_task)
+        db.session.commit()
+        new_task_response = {
+            "task": {
+                "id": new_task.task_id,
+                "title": new_task.title,
+                "description": new_task.description,
+                "is_complete": False if new_task.completed_at == None else True
+            }
+        }
```

Comment:
Another good place to use a task dictionary helper method here. For the sake of making repetitive comments, anywhere you'll be returning a dictionary of a task.